Update word count chart

maddiecomtois opened the pull request:
- Made the word count chart compatible with the classroom selection option.
- Used the first classroom as the default when opening the dashboard
- Added a message to teachers for those who do not have any classrooms
- (Didn't delete the carousel code quite yet in case we want to use it for something else later)

Need a translation for: _Add students to a classroom to view their statistics_

<img width="1107" alt="Screen Shot 2022-10-19 at 14 37 48" src="https://user-images.githubusercontent.com/13711456/196791127-9a283c9a-ca51-4cf9-a37e-1ce089983368.png">

maddiecomtois commented:
First attempt at customizing the CSS for the chart legend.  Still a lot of work to be done and not everything is working yet, so please continue if you so wish

<img width="617" alt="Screen Shot 2022-10-19 at 16 52 57" src="https://user-images.githubusercontent.com/13711456/196811991-072e962d-4060-4783-9418-651b35821636.png">

OisinNolan commented:
- Here's the latest CSS, I think this should do for the time being. 
- Next up: add time filter to the classroom selector component

<img width="809" alt="image" src="https://user-images.githubusercontent.com/34962541/197021964-598fbf4c-f499-41b1-9179-fa41d2baad12.png">

maddiecomtois commented:
- Added a date picker to the ```classroom-selector``` component
- Changed the return object from this component to return an object {classroom, startDate, endDate} instead of just the selected classroom
- Doesn't do anything yet with the dates, just prints them to the console

<img width="925" alt="Screen Shot 2022-10-20 at 19 16 03" src="https://user-images.githubusercontent.com/13711456/197082898-764147fb-1f3c-43f7-a867-e57a6d6c52c0.png">
<img width="969" alt="Screen Shot 2022-10-20 at 19 15 37" src="https://user-images.githubusercontent.com/13711456/197082915-2b75e233-7447-4e9a-a112-7c8686ba9fbe.png">
<img width="898" alt="Screen Shot 2022-10-20 at 19 15 49" src="https://user-images.githubusercontent.com/13711456/197082920-dd2a8e97-28ea-4e8e-803e-099eb3f6b35a.png">
